.release-please-manifest.json

```diff
@@ -22,5 +22,6 @@
   "plugins/source/terraform": "0.5.6",
   "plugins/source/terraform+FILLER": "0.0.0",
   "plugins/source/test": "1.1.4",
-  "plugins/source/test+FILLER": "0.0.0"
+  "plugins/source/test+FILLER": "0.0.0",
+  "plugins/source/heroku": "1.0.0"
```

Comment:
We should add a filler entry for `heroku` before merging to avoid conflicts between release PRs